db/migrate/20160825122452_remove_duplicate_drafts.rb

```diff
@@ -0,0 +1,34 @@
+class RemoveDuplicateDrafts < ActiveRecord::Migration
+  def up
+    content_items = ContentItem.where(id: [815434, 818621])
+
+    if content_items.any? { |x| x.updated_at != x.created_at }
+      raise "content item #{x.id} has been changed"
```

Comment:
I think this should fail silently here as if you ran through all migrations you wouldn't want this to die here.